src/crimson/osd/pg.cc

```diff
@@ -736,11 +754,15 @@ seastar::future<Ref<MOSDOpReply>> PG::do_osd_ops(
       *m,
       obc->obs.oi.soid);
     return seastar::make_ready_future<Ref<MOSDOpReply>>(std::move(reply));
-  }, osd_op_errorator::all_same_way([ox = ox.get(),
+  }, OpsExecuter::osd_op_errorator_no_eagain::all_same_way([ox = ox.get(),
                                      m,
                                      obc,
                                      this] (const std::error_code& e) {
     return handle_failed_op(e, std::move(obc), *ox, *m);
+  }), PGBackend::read_errorator::all_same_way([m,
```

Comment:
Why do we providing a handler for all errors of `PGBackend::read_errorator` here? It extends `ll_read_errorator` which in turn is an alias to following errorator defined in `FuturizedStore`:

```cpp
  using read_errorator = crimson::errorator<crimson::ct_error::enoent,
                                            crimson::ct_error::input_output_error>;
```

I doubt we really would like to take care of `enoent` in this way. I guess it might work by "an accident" as `enoent` a part of of the `osd_op_errorator_no_eagain`.
Hmm, perhaps `errorator` should be extended to catch problems like this one.